Names.md

```diff
@@ -27,6 +27,6 @@ This is a document holds the team members of the different sub teams in the A2SV
 * **Full-Time**
   * Full Name
 * **Part-Time**
-  * Full Name
+  * Bereket Heramo DTL 
```

Comment:
Sorry, my mistake but we have decided to not include the project in the names so please remove that